pypit/flatfield.py

```diff
@@ -34,6 +34,10 @@ def default_settings():
                         )
     return default_settings
 
+# TODO, JFH: I do not understand why ArcImage is its own class whereas there is no class called FlatImage. In principle
```

Comment:
yes, but there may be separate images for slit tracing and pixel flat fielding (e.g. LRIS)

Reply:
Let's discuss this during the next telecon.